src/main/java/com/epam/repository/PersonDAO.java

```diff
@@ -32,4 +32,52 @@ Person buildPerson(ResultSet rs) throws SQLException {
                 .build();
     }
 
+    public boolean doesEmailExists(String email) {
+        List<String> emailsList = jdbcTemplate.query("SELECT email FROM person", (rs, rowNum) -> getStringEmail(rs));
+        for(String stringEmail:  emailsList){
+            if (stringEmail.equals(email)) return true;
+        }
+        return false;
+    }
+
+    String getStringEmail(ResultSet rs) throws SQLException {
+        return rs.getString("email");
+    }
+
+    public boolean addPersonToDataBase(Person person) {
```

Comment:
Please call methods simpler. Just addPerson. It's obvious we add it in DB if this method located in DAO

Reply:
I think would be the better return Person, not boolean

Reply:
Done